src/plugins/syscalls/syscalls.cpp

```diff
@@ -149,6 +244,21 @@ static GSList* create_trap_config(drakvuf_t drakvuf, syscalls *s, symbols_t *sym
 
             PRINT_DEBUG("[SYSCALLS] Adding trap to %s\n", symbol->name);
 
+            syscall_wrapper_t *wrapper = (syscall_wrapper_t *)g_malloc(sizeof(syscall_wrapper_t));
+
+            wrapper->syscall_index = -1;
```

Comment:
Should move `wrapper->sc=s` here so that the callback would have access to the class even if the index is not found.